tsconfig-es.json

```diff
@@ -1,10 +1,11 @@
 {
 	"extends": "./tsconfig-base.json",
 	"compilerOptions": {
-		"module": "es6",
+		"module": "CommonJS",
```

Comment:
Why are we making this change? Will this be a breaking change?

Reply:
This change is made because the build failed when trying to import the  form-data module. This worked along with esModuleInterop being set true.

Reply:
I suspect that we don't want to make this change. We have a tsconfig for ES6 and CommonJS. I don't think we want the `import FormData from "form-data";` in GraphRequest. FormData is available in most browsers. It will be up to the customer to import formdata when running on Node. You should add an update to the readme to provide instuction that they need to install and import  formdata if they are running on Node and they need to make calls to Microsoft Graph that accept formdata.

Reply:
Restored the tsconfig files